### ISSUE TITLE
Add ability to filter by Elm version

### ISSUE THREAD

**Issue by sdenier** [elm-community/builtwithelm#109]
Elm seems to change quite rapidly from version to version (for example 0.17 -> 0.18). It would be great to be able to filter examples by Elm version, then one can lookup how other people uses the latest versions. Especially for beginners, this could be of great help to be able to find quickly recent examples, as the tutorials are not always complete or up to date.

**Comment by joelclermont** [comment]
I like this idea, but I wonder if we would end up with out of date metadata. For example, someone adds a project when they first write it, say in 0.18. But will they remember to come back and update this repo when 0.19 comes out months later and they update their project?

**Comment by sdenier** [comment]
Indeed, that would be a problem for people looking for old examples and then stumbling upon updated versions.

But I think the primary use case would be beginners, and they should be mostly interested by examples with the most recent version. Instead of providing a search field, a simple checkbox "Show me examples with the latest Elm version" would suffice?

- one declares a project with version Elm 0.18
- the website can filter all projects from the latest version, namely 0.18
- later, Elm 0.19 is released and the website is updated to mirror that
- projects declared as 0.18 no longer appear as "recent", unless manually updated.

Sure, we would miss some update but at least, there is less false positive results.

Next step of course would be to retrieve `elm-version` from the elm-package.json :)
